WhatsAppNotification/VirtoCommerce.WhatsAppNotification.Web/module.manifest

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<module>
+  <id>VirtoCommerce.WhatsAppNotification</id>
+  <version>1.0.2</version>
+
+  <platformVersion>2.13.47</platformVersion>
+  <dependencies>
+  </dependencies>
+
+  <title>WhatsApp notification module</title>
+  <description>This module provide WhatsApp notification gateway</description>
```

Comment:
Sample sending Order change notifications to WhatsApp via Twilio or Woowa gateways